CPT331.iOS/CPT331.iOS/String.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  String.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 20/09/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    // Third party code: http://stackoverflow.com/a/32512122
```

Comment:
The only time you actually have to know how to solve something yourself is when you're in an interview - the rest of the time everyone just nicks stuff from SO :-)